conf/kpl.properties

```diff
@@ -187,7 +187,7 @@ MetricsUploadDelay = 60000
 # Default: 1
 # Minimum: 1
 # Maximum (inclusive): 16
-MinConnections = 8
+MinConnections = 1
```

Comment:
Where did this come from?

Reply:
Description of this config:

```
# Minimum number of connections to keep open to the backend.
#
# There should be no need to increase this in general.
#
```

Although I don't fully understand the reason, but it does seem recommended to keep it as default.